consensus/ConsensusFileGenerator.py

```diff
@@ -108,42 +108,42 @@ def _add_history_of_variant(id_to_match, export, variant_history):
-        old_id = '{}_{}_{}_{}_{}'.format(chromosome, position, ref, alt, gene)
+        old_id = ConsensusFileGenerator._get_variant(chromosome, position, ref, alt, gene)
         ids.append(old_id)
         # This is done for the variants that lack their anchor (before the 2019 okt export)
         if variant_type == "del":
             old_ref = ref[1::]
             old_alt = '.'
             old_pos = int(position) + 1
-            old_id_del_ins = '{}_{}_{}_{}_{}'.format(chromosome, old_pos, old_ref, old_alt, gene)
+            old_id_del_ins = ConsensusFileGenerator._get_variant(chromosome, old_pos, old_ref, old_alt, gene)
             ids.append(old_id_del_ins)
         elif variant_type == "ins" or variant_type == "dup":
             old_ref = "."
             old_alt = alt[1::]
-            old_id_del_ins = '{}_{}_{}_{}_{}'.format(chromosome, position, old_ref, old_alt, gene)
+            old_id_del_ins = ConsensusFileGenerator._get_variant(chromosome, position, old_ref, old_alt, gene)
             ids.append(old_id_del_ins)
         elif variant_type == "delins":
             # ids for indels when this was still an issue https://github.com/molgenis/data-transform-vkgl/issues/12
             old_pos = str(int(position) - 1)
-            old_ref1 = "A" + ref
-            old_alt1 = "A" + alt
-            old_id1 = Hasher.hash(chromosome + '_' + old_pos + '_' + old_ref1 + '_' + old_alt1 + '_' + gene)[0:10]
-            old_ref2 = "G" + ref
-            old_alt2 = "G" + alt
-            old_id2 = Hasher.hash(chromosome + '_' + old_pos + '_' + old_ref2 + '_' + old_alt2 + '_' + gene)[0:10]
-            old_ref3 = "T" + ref
-            old_alt3 = "T" + alt
-            old_id3 = Hasher.hash(chromosome + '_' + old_pos + '_' + old_ref3 + '_' + old_alt3 + '_' + gene)[0:10]
-            old_ref4 = "C" + ref
-            old_alt4 = "C" + alt
-            old_id4 = Hasher.hash(chromosome + '_' + old_pos + '_' + old_ref4 + '_' + old_alt4 + '_' + gene)[0:10]
+            old_id1 = ConsensusFileGenerator._get_hashed_old_variant('A', chromosome, old_pos, ref, alt, gene)
+            old_id2 = ConsensusFileGenerator._get_hashed_old_variant('G', chromosome, old_pos, ref, alt, gene)
+            old_id3 = ConsensusFileGenerator._get_hashed_old_variant('T', chromosome, old_pos, ref, alt, gene)
+            old_id4 = ConsensusFileGenerator._get_hashed_old_variant('C', chromosome, old_pos, ref, alt, gene)
             ids.append(old_id1)
             ids.append(old_id2)
             ids.append(old_id3)
             ids.append(old_id4)
-
         return ids
 
+    @staticmethod
+    def _get_variant(chromosome, pos, ref, alt, gene):
+        return f'{chromosome}_{pos}_{ref}_{alt}_{gene}'
+
+    @staticmethod
+    def _get_hashed_old_variant(anchor, chromosome, pos, ref, alt, gene):
+        return Hasher.hash(
+            ConsensusFileGenerator._get_variant(chromosome, pos, f'{anchor}{ref}', f'{anchor}{alt}', gene))[0:10]
```

Comment:
Same here, a non-static function would just be able to use self instead of ConcensusFileGenerator